### ISSUE TITLE
🐛 Bug: Restricted in strict mode error

### ISSUE THREAD

**Issue by punndcoder28** [nocodb/nocodb#4312]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

I installed the `nocodb-sdk` and when trying to initialise the module by importing `import { Api } from 'nocodb-sdk'`, it throws an error saying `Restricted in strict mode`. 

Steps:

1. `npm i nocodb-sdk`
2. `import { Api } from 'nocodb-sdk'`

### Desired Behavior

The `Api` object should imported successfully allowing us to initialise the project and get data from the tables

### Project Details

```
Node: **v12.22.12**
Arch: **x64**
Platform: **linux**
Docker: **false**
Database: **pg**
ProjectOnRootDB: **false**
RootDB: **pg**
PackageVersion: **0.98.1**
```

### Attachements

![Screenshot_20221108-122802](https://user-images.githubusercontent.com/32433964/200496224-d24e5fee-5e1f-4373-9f07-c351de850eb3.png)

**Comment by punndcoder28** [comment]
There seems to be similar reports from [axios library](https://github.com/axios/axios/issues/4998) but can confirm that even after trying the suggested fixed mentioned on that thread, I was not able to resolve the issue

**Comment by wingkwong** [comment]
I couldn't produce the issue on the latest version (0.98.4). Can you confirm it on your machine?

**Comment by punndcoder28** [comment]
<img width="1384" alt="Screenshot 2022-11-16 at 5 29 42 PM" src="https://user-images.githubusercontent.com/32433964/202175291-dedea7ba-1e75-46eb-9afb-a3d3984bc954.png">

I am using the following package versions
```
nocodb-sdk: ^0.98.4
axios: ^1.1.3
```

Could you let me know your dependency versions? As soon as I disable hermes, the project works as expected.

**Comment by wingkwong** [comment]
@punndcoder28 

In `nocodb-sdk`, we are using `axios@^0.21.1`.  Is there any reason to use `axios: ^1.1.3`? It seems that the error thrown from axios since 1.0.0. If it's really necessary to use a latest version of axios. They've released `axios@1.2.0-alpha.1` which resolved the problem. Can you help confirm if bumping to `axios@1.2.0-alpha.1` resolves the problem?

**Comment by punndcoder28** [comment]
Sorry for the late reply. Tried with `axios@1.2.0-alpha.1` but to no avail unless I disable hermes in the project for android. We started off with `axios: 1.0.0` and downgrading the dep might cause other features on the project to break. As a result I am refraining from doing that.

**Comment by wingkwong** [comment]
Got it. Could you share how did you disable `hermes`? and have you removed `node_modules` before trying `axios@1.2.0-alpha.1`?

**Comment by punndcoder28** [comment]
Yes. I deleted `node_modules` before installing the alpha version for axios. To disable hermes, I did so in the `android/app/build.gradle` file.

**Comment by punndcoder28** [comment]
Hi @wingkwong , any updates on the issue?

**Comment by wingkwong** [comment]
`hermes` is a JS engine for React Native applications. Some people have pointed out it is due to `axios`. If the root cause is from `axios`, there is not much we can do. I've seen they released `v1.2.0`. You may give it a try.

**Comment by dstala** [comment]
closing this thread. if you are still facing issue, help us with more details.
